dask/utils.py

```diff
@@ -2064,3 +2064,20 @@ def maybe_pluralize(count, noun, plural_form=None):
         return f"{count} {noun}"
     else:
         return f"{count} {plural_form or noun + 's'}"
+
+
+def is_namedtuple_instance(obj: Any) -> bool:
+    """Returns True if obj is an instance of a namedtuple.
+
+    Note: This function checks for the existence of the methods and
+    attributes that make up the namedtuple API, so it will return True
+    IFF obj's type implements that API.
```

Comment:
Nit: I don't think iff is accurate here, I could easily create an object that has these attributes and is not a `namedtuple`.

Reply:
This might be arguing over semantic details but in that case your objects _implements_ the `namedtuple` API. I actually tried to word this carefully to NOT say that the object _is_ a `namedtuple`. LMK what you think :)

Reply:
Definitely semantic, I don't expect it to come up. But there's no guarantee that these attributes do the same thing as the `namedtuple` APIs.